puppet/modules/slave/manifests/unittests.pp

```diff
@@ -168,4 +168,12 @@
 
   # RVM
   include slave::rvm
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '9' {
+    include slave::rvm
+  }
```

Comment:
```suggestion
  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '9' {
    include slave::rvm
  }
```

Otherwise you always include `rvm` ;)

Reply:
Touche